docs/tabpy-tools.md

```diff
@@ -265,6 +265,48 @@ tabpy.query('Sentiment Analysis', _arg1, library='textblob')[‘response’]
 
 ```
 
+### T-Test
```

Comment:
Run `markdown-toc` to update TOC for the document.

Reply:
fixed